Samples/Tutorials/Embeddings/index.ts

```diff
@@ -75,7 +75,7 @@ const client = Client
                 console.log(`Removed dish counter: ${counter}`);
                 await client.embeddings
                     .forTenant(TenantId.development)
-                    .remove(DishCounter, counter.dish);
+                    .resolveDeletionToEvents(DishCounter, counter.dish);
```

Comment:
Oh yeah forgot to think about this case, shieeet. Idea was that this method is still called `delete()`, it's just the embeddings (as in the class/builder) that has the `resolveXToEvents()`. But `delete` is a reserved keyword in JS (which is why we changed to `remove` in the first place). Let's think about this.

Reply:
Or wait, maybe this works as we aren't declaring a global function (like with the decorator), and this method can be still called `delete`

Reply:
Did a quick test and yes, `client.embeddings.forTenant().delete()` works, so let's rename this method to `delete`. Very happy about that actually, we get super consistent API 👍

Reply:
Done.